reco_utils/recommender/deeprec/io/nextitnet_iterator.py

```diff
@@ -0,0 +1,501 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import tensorflow as tf
+import numpy as np
+import json
+import pickle as pkl
+import random
+import os
+import time
+
+from reco_utils.recommender.deeprec.deeprec_utils import load_dict
+
+__all__ = ["NextItNetIterator"]
+
+
+class NextItNetIterator:
```

Comment:
inherit the SequentialIterator and reuse some functions:
class NextItNetIterator(SequentialIterator):

Reply:
Now NextitnetIterator inherit SequentialIterator and reuse most of the funcitons